revolv/static/css/screen.css

```diff
@@ -5783,8 +5957,74 @@ header.edit-project-header > .container > h1 {
     font-size: 11.57px;
     line-height: 21px;
   }
+
+/* 30-Jun */
+
+.contents .details-active-project-module .banner-section .funded-round {
+  bottom: -767px;
+}
+.contents .details-active-project-module .banner-section .container {
+  min-height: auto;
+}
+.details-active-project-module .video-main {
+  display: block;
+  overflow: hidden;
+  padding-bottom: 45px;
+  width: auto;
+}
+.details-active-project-module .info-section .container {
+  padding: 4px 83px 0 92px;
+}
+.details-active-project-module .container {
+  padding: 4px 37px 0 59px;
+}
+.project-updates-module .container {
+  padding: 38px 39px 20px 62px;
+  position: relative;
+  width: 100%;
+}
+.details-active-project-module.title-white-border {
+  margin-right: 15px;
+}
+.venue-area {
+  text-align: center;
+  width: 100%;
+}
+.main-area1 .venue-area h2 {
+  display: inline-block;
+  float: none !important;
+  font-size: 19px;
+  height: auto;
+  margin-top: 40px;
+  padding: 7px 45px 6px;
+  line-height: normal;
+}
+.project-updates-module .right-aside {
+  width: 100%;
 }
+.footer .bottoms ul.social-ul li .icon-g-plus.icons {
+  background-position: -80px -12px;
+}
+.footer .bottoms ul.social-ul li:hover .icon-g-plus.icons {
+  background-position: -100px -12px;
+}
+
```

Comment:
Please leave only one blank line here. It would probably be helpful to have a comment next to the closing brace on line 6016 to say that that's the end of the `@media` block.